primenet.py

```diff
@@ -5207,7 +5211,7 @@ def get_assignments(adapter, adir, cpu_num, progress, tasks):
             tf_min = ""
             tf_max = ""
             if msec_per_iter is not None:
-                ghd_to_request = int((days_work.total_seconds() - time_left.total_seconds()) * 1000 / msec_per_iter)
+                ghd_to_request = int(max(10,(days_work.total_seconds() - cur_time_left)) * 1000 / msec_per_iter)
```

Comment:
Unfortunately, `.total_seconds()` requires Python 3.2+. It probably would be easier to just use the original value directly:
```py
ghd_to_request = max(10, ((options.days_of_work * 24 * 60 * 60) - cur_time_left) * 1000 / msec_per_iter)
```